packages/nodes-base/nodes/HttpRequest/test/utils/utils.test.ts

```diff
@@ -136,5 +136,41 @@ describe('HTTP Node Utils', () => {
 				uri: 'https://example.com',
 			});
 		});
+		it('should redact the Authorization header', () => {
+			const requestOptions: IRequestOptions = {
+				method: 'POST',
+				uri: 'https://example.com',
+				body: { sessionToken: 'secret', other: 'foo' },
+				headers: { authorization: 'Bearer some-sensitive-token', other: 'foo' },
```

Comment:
Maybe also a test case where the header is with a uppercase first letter `Authorization`?